CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## [Unreleased]
 
+## [0.1.2-alpha.2] - 2017-11-28
+- Added "observer" attribute to @property annotation
```

Comment:
Can you move both items to the `unreleased` section and remove these alpha releases?